ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -504,6 +504,18 @@ def test_source_ipahealthcheck_meta_core_metacheck(self):
         assert data[0]["kw"]["ipa_version"] in result.stdout_text
         assert data[0]["kw"]["ipa_api_version"] in result.stdout_text
 
+    def test_fips_mode_check_exists(self):
+        """
+        Testcase to verify whether FIPS mode check exists
+        in ipahealthcheck.meta.core
+        """
+        returncode, data = run_healthcheck(self.master,
+                                           "ipahealthcheck.meta.core")
```

Comment:
Currently there is only one check in "ipahealthcheck.meta.core" but this could change in the future. You can restrict the checks to "MetaCheck" only.